azure-proto-compute/AvailabilitySetContainer.cs

```diff
@@ -27,14 +27,17 @@ internal AvailabilitySetContainer(AzureResourceManagerClientOptions options, Res
         /// Initializes a new instance of the <see cref="AvailabilitySetContainer"/> class.
         /// </summary>
         /// <param name="options"> The client parameters to use in these operations. </param>
-        /// <param name="id"> The identifier of the resource that is the target of operations. </param>
-        internal AvailabilitySetContainer(AzureResourceManagerClientOptions options, ResourceIdentifier id)
-            : base(options, id)
+        /// <param name="parentId"> The identifier of the resource that is the target of operations. </param>
```

Comment:
use the parentId string from https://github.com/m-nash/azure-proto-sdk/blob/master/XmlDocStandard.md#commonly-used